oshop-dashboard/src/app/app.component.html

```diff
@@ -0,0 +1,516 @@
+
```

Comment:
clear this file and just add <div>app</div>. This is all generated garbage from the angular cli. We should remove it since we don't need it